TextMate/TextMate2.download.recipe

```diff
@@ -3,75 +3,85 @@
 <plist version="1.0">
 <dict>
     <key>Description</key>
-    <string>Downloads the latest TextMate 2.
+    <string>Downloads the latest TextMate 2 from their GitHub repo.
 
-BRANCH can be one of 'release' or 'beta', which correspond respectively
-to the 'Normal' and 'Nightly' options in TextMate 2's Software Update
-preferences panel.
-
-MACOS corresponds to the macOS version. TextMate API currently offers options
-from 10.12 to 10.latest.
-</string>
+Set PRERELEASE to a non-empty string to download prereleases, either
+via Input in an override or via the -k option,
+i.e.: `-k PRERELEASE=yes`</string>
     <key>Identifier</key>
     <string>com.github.autopkg.download.TextMate2</string>
     <key>Input</key>
     <dict>
-        <key>BASE_URL</key>
-        <string>https://api.textmate.org/downloads</string>
         <key>NAME</key>
         <string>TextMate2</string>
-        <key>BRANCH</key>
-        <string>release</string>
-        <key>MACOS</key>
-        <string>10.15</string>
-        <key>FILENAME</key>
-        <string>TextMate_2.0.tbz</string>
+        <key>PRERELEASE</key>
+        <string></string>
     </dict>
     <key>MinimumVersion</key>
     <string>1.3</string>
     <key>Process</key>
     <array>
-        <dict>
-            <key>Arguments</key>
-            <dict>
-                <key>filename</key>
-                <string>%FILENAME%</string>
-                <key>url</key>
-                <string>%BASE_URL%/%BRANCH%?os=%MACOS%</string>
-            </dict>
-            <key>Processor</key>
-            <string>URLDownloader</string>
-        </dict>
+		<dict>
+			<key>Processor</key>
+			<string>GitHubReleasesInfoProvider</string>
+			<key>Arguments</key>
+			<dict>
+				<key>github_repo</key>
+				<string>textmate/textmate</string>
+				<key>include_prereleases</key>
+				<string>%PRERELEASE%</string>
+			</dict>
+		</dict>
```

Comment:
What's the purpose of calling this processor twice? The second one has more arguments - what exactly does this one do?

Reply:
That's my bad, sorry about that. Should be corrected now